### DISCUSSION TITLE
Option to automatically turn on Autoplay

### DISCUSSION THREAD

**Discussion by 4RK0S4H4** [ciderapp/Cider#1427] [Feature Request]
Hello, I think a nice feature to add would be an option to turn on the Autoplay queue on launch. I personally always keep it on so when the songs I have queued ends I still have some music playing afterwards.

![image](https://user-images.githubusercontent.com/84862166/189946192-78354442-00bc-42b6-ae2f-6bfc7aa992d5.png)

In its current state Autoplay is turned off on launch regardless of whether you had it on or off in a previous instance on Cider or on your phone

**Comment by permster** [comment]
I pretty much just leave the app on a playlist shuffle in the systray all day long and this annoys me when I forget to enable autoplay.

**Reply by 4RK0S4H4** [reply]
This issue was fixed for me many months ago btw, now it keeps the autoplay setting even after relaunching, using latest Microsoft Store version of Cider btw

**Reply by permster** [reply]
I'm on the Microsoft Store version 2.1.0 and this is still an issue for me.  If I leave the app open and switch between playlists it's fine but once I exit.  Open it back up and hit shuffle on a playlist, it starts playing but autoplay is off.
